catalog/pipeline/fbc-release/0.11/README.md

```diff
@@ -0,0 +1,74 @@
+# FBC Release Pipeline
+
+ Tekton release pipeline to interact with FBC Pipeline                       
+
+## Parameters
+
+| Name | Description | Optional | Default value |
+|------|-------------|----------|---------------|
+| snapshot | The Snapshot in JSON format | No | - |
+| enterpriseContractPolicy | JSON representation of the EnterpriseContractPolicy | No | - |
+| fromIndex | Index image (catalog of catalogs) the FBC fragment will be added to | No | - |
+| targetIndex | Index image (catalog of catalogs) the FBC fragment will be added to | No | - |
+| binaryImage | OCP binary image to be baked into the index image | Yes | "" |
+| buildTags | List of additional tags the internal index image copy should be tagged with | Yes | "[]" |
+| addArches | List of arches the index image should be built for | Yes | "[]" |
+| requester | Name of the user that requested the signing, for auditing purposes | No | - |
+| signingConfigMapName | The ConfigMap Name required by the Pipeline | Yes | "hacbs-signing-pipeline-config" |
+| fbcPublishingCredentials | Secret used to publish the built index image | Yes | "fbc-publishing-credentials" |
+| requestUpdateTimeout | Max seconds to wait until the status is updated | Yes | - |
+| buildTimeoutSeconds | Max seconds to wait until the build finishes | Yes | - |
+
+## Changelog
+
+### Changes since 0.10
+- updates tasks that uses `create-internal-request` task to 0.6
+- `publish-index-image` now uses `create-internal-request`
+- only executes `public-index-image` when extraData result has `fbc_opt_in=true` set
```

Comment:
`publish` instead of `public`?

Reply:
Also, result from what? The `add-fbc-contribution-to-index-image` task, right? Can we add that?

Reply:
Ok. Some more info added.